.ci/scripts/test_llama.sh

```diff
@@ -84,7 +91,7 @@ cmake_build_llama_runner() {
         -DEXECUTORCH_BUILD_CUSTOM="$CUSTOM" \
         -DEXECUTORCH_BUILD_OPTIMIZED=ON \
         -DEXECUTORCH_BUILD_XNNPACK="$XNNPACK" \
-        -DEXECUTORCH_BUILD_OPTIMIZED=ON \
+        -DEXECUTORCH_BUILD_QUANTIZED="$QE" \
```

Comment:
Can we not always build? haveing so many options for build feels like additional burden for users. Maybe do default opt-in?